neurom/fst/_neuritefunc.py

```diff
@@ -81,6 +81,14 @@ def n_leaves(neurites, neurite_type=NeuriteType.all):
     return n_sections(neurites, neurite_type=neurite_type, iterator_type=Tree.ileaf)
 
 
+def total_surface_area_per_neurite(neurites, neurite_type=NeuriteType.all):
+    '''Surface area in a collection of neurites.
+
+    The area is defined as the sum of the area of the sections.
+    '''
+    return [neurite.area for neurite in iter_neurites(neurites, filt=is_type(neurite_type))]
+
+
```

Comment:
Sadly, we already have inconsistencies.  I think `total_area_per_neurite` makes sense